src/components/Common/MiddleTruncatedText/index.tsx

```diff
@@ -15,23 +17,47 @@ type Props = {
   endLength?: number;
   textProps?: TextProps;
   placement?: PlacementWithLogical;
+  isCopy?: boolean;
 };
 const MiddleTruncatedText = ({
   value,
   startLength = 5,
-  endLength = 4,
+  endLength = 7,
   textProps,
   placement,
+  isCopy,
 }: Props) => {
+  const { t } = useTranslation();
+  const { onCopy, hasCopied } = useClipboard(value || '');
+
+  const handleOnClick = () => {
```

Comment:
Minor: I think this component is quite common and can be placed in many places which already has an `onClick` handler. To prevent bubbling or delegation or weird bugs we should specifically use `event.preventDefault()` and `event. stopPropagation()` for this one.

Reply:
Hi @duc-q ,
For text, it won't have an event from the standard event of the browser (Ref: https://developer.mozilla.org/en-US/docs/Web/API/Event)
For `stopPropagation`, the root cause will be a rarity structure call event from "parent", so we must use it to handle this event. But for the current structure, we don't need to use it.  Another side, we can put it later if we have a use case which avoids process event from the parent